sechub-doc/src/docs/asciidoc/documents/shared/infrastructure/products/07_03_98_infra_security_products_prepare.adoc

```diff
@@ -0,0 +1,15 @@
+// SPDX-License-Identifier: MIT
+[[section-infrastructure-setup-security-products-prepare]]
+==== Prepare (Remote Data Preparation)
+
+`Prepare` is not a scan product, but a developed wrapper application to prepare remote sources for scans.
+It is used to download remote sources and upload them to the shared storage make them available to the scan products.
+This enables the user to scan remote data without the need to store them local on a machine an upload it with the client.
```

Comment:
```suggestion
This allows the user to scan remote data without having to store it locally on a computer and upload it with the client.
```
At the beginning I found only a "machine an upload" which has to be an "machine and upload", but then I rewrote the sentence also a bit.